R/archivePackages.R

```diff
@@ -19,36 +21,39 @@
 ##'   archivePackages(pkg = "drat")  # archive older copies of just one package
 ##' }
 ##' @author Thomas J. Leeper
-archivePackages <- function(repopath=getOption("dratRepo", "~/git/drat"),
-                            type="source", 
+archivePackages <- function(repopath = getOption("dratRepo", "~/git/drat"),
+                            reldir = "src/contrib",
+                            type = "source", 
                             pkg) {
```

Comment:
I would ask to move the new argument `reldir="src/contrib"` to the _end_ of the argument list.  While R has named arguments, folks sometimes use positional placement as a shortcut so the the very common convention is to grow at the end.

Please change that.

Reply:
hardcoding path separator might not be best idea. Does this change work on windows?

Reply:
Yes. We could do `reldir = file.path("src", "contrib")` or even add a (local) one-line helper a la (untested)

```r
.getContribDir <- function() file.path("src", "contrib")
```

to open the door to more testing etc pp

Reply:
Hi Dirk, not sure if I done all this PR stuff correctly. Please be tolerant - this is my 1st PR on Github :-)

Reply:
Absolutely :+1: Will help as much as possible.  This was already a very good PR (the lack of a prior heads-up notwithstanding) so I am sure we can shepherd this.